backend/GymBeam/Controllers/UserController.cs

```diff
@@ -99,18 +97,53 @@ public async Task<ActionResult<bool>> CheckUsernameAvailability(string username)
 #if !DEBUG
         [Authorize(Roles = Roles.Admin)]
 #endif
-        public IActionResult ChangeRole(int id, string role)
+        public async Task<IActionResult> ChangeRole(int id, string role)
         {
-            return NoContent();
+            if (role != "User" && role != "Admin")
```

Comment:
How can I not hardcode this here if our "constant" roles are only used for JWT authorization and I can't just use Roles.Admin and Roles.User?

Reply:
this check should be in validator. It should compare to Domain.Role